src/lib/validationRules/noMixedRedirectUris/noMixedRedirectUrls.ts

```diff
@@ -18,6 +18,7 @@
 // OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
 // SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 //
+import { isHostnameLocal, isUriLocalhost } from "../../helperFunctions";
```

Comment:
Please fix the lint warning

Reply:
done with 81b6eabeda8b46519ee57062a77cb0f5613189e3